packages/tools/kadena-cli/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "outDir": "./lib",
     "lib": ["ES2022"],
     "module": "NodeNext",
-    "types": [".kadena/pactjs-generated", "./src/utils/ttys.d.ts"]
+    "types": [".kadena/pactjs-generated", "./src/utils/ttys.d.ts"],
+    "resolveJsonModule": true
```

Comment:
(node:21486) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

zie import package.json in the pactjservice.

any suggestions @barthuijgen

Reply:
Not sure what the implications are for compatibility. To be safe I think just I'd use `fs.readFile()`.